packages/web-react/src/components/FileUploader/FileUploaderAttachment.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { createRef, RefObject } from 'react';
+import classNames from 'classnames';
+import { SpiritFileUploaderAttachmentProps } from '../../types';
+import { useStyleProps } from '../../hooks';
+import { useFileUploaderStyleProps } from './useFileUploaderStyleProps';
+import { useFileUploaderAttachment } from './useFileUploaderAttachment';
+import AttachmentDismissButton from './AttachmentDismissButton';
+import { Icon } from '../Icon';
+
+const FileUploaderAttachment = (props: SpiritFileUploaderAttachmentProps) => {
+  const { label, name, id, onDismiss, buttonLabel = 'Remove', file, onError, ...restProps } = props;
+
+  const { classProps } = useFileUploaderStyleProps();
+  const { styleProps, props: transferProps } = useStyleProps(restProps);
+
+  const attachmentRef = createRef() as RefObject<HTMLLIElement>;
```

Comment:
Not necessary right now. Just for info - https://react.dev/reference/react/createRef

`createRef` is mostly used for [class components.](https://react.dev/reference/react/Component) Function components typically rely on [useRef](https://react.dev/reference/react/useRef) instead.

```suggestion
  const attachmentRef = useRef() as RefObject<HTMLLIElement>;
```

Reply:
Fixed